lib/board.rb

```diff
@@ -3,43 +3,49 @@ class Board
   def initialize
     @grid = Array.new(3) { Array.new(3, '_') }
   end
+
   def [](position)
     row, col = position
     @grid[row][col]
   end
+
   def []=(position, value)
     row, col = position
     @grid[row][col] = value
   end
```

Comment:
Kindly change this to use attr_reader/attr_writer, as this defeats the purpose of using attrs, refer to [this stack overflow post](https://stackoverflow.com/a/5046915/12217655) for tips, also [this article](https://www.rubyguides.com/2018/11/attr_accessor/) discusses how attr_reader/attr_writer is a shorter syntax.